### PR TITLE
Add initial LJ fitting function 

### PR DESCRIPTION
**Pull request by bzhang25**
Least square function should be working.
Still need to figure out how to call the scf and mp2 energies and incorporate wavefunction classes

### CODE REVIEW

**Comment by bzhang25** [comment]
Also need to add tests

**Comment by dgasmith** [comment]
It looks like you have both a `project` and `quest` folders in here. Can you please delete the `project` folder.

**Comment by dgasmith** [comment]
Ill pull that in once thats done, please work on adding good docs and the like after this is through.

**Comment by bzhang25** [comment]
Done. 
We will work on documentation from now on

File: quest/lj.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import numpy as np
+import molecule
```
Comment (dgasmith) [inline quest/lj.py]: Make sure this is relative import `from . import molecule`.

File: quest/lj.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    A,B = get_coeffs(distances, energies)
+    # calculate sigma
+    sigma = np.power(A/(-B), 1.0/6.0)
+    # returning the correct values
+    if(returnAll):
+        return sigma, A, B, energies, distances
+    else:
+        return sigma
+
+def get_coeffs(distances, energies):
```
Comment (dgasmith) [inline quest/lj.py]: I might call this `fit_lj` instead.

File: quest/lj.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
+import numpy as np
+import molecule
+import psi4
+
+# .mol returns a psi4 geometry
+# setmol can be given a string and it sets up a molecule
+
+# atom - Input molecule (should be one atom)
+# returnAll - Returns coefficients and energies/distances in addition to sigma if True
+def lj_fit(mol, returnAll=False):
```
Comment (dgasmith) [inline quest/lj.py]: `this probably should be `build_lj_coefficients` or the like.

File: quest/lj.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    perform a list squares fitting and 
+    returns coefficients A for the r^12 term and B for the r^6 term
+    """
+
+    powers = [-12.0, -6.0]
+    r_power = np.power(np.array(distances).reshape(-1,1), powers) 
+    A,B = np.linalg.lstsq(r_power, energies)[0]
+    return A,B
+
+if __name__=='__main__':
```
Comment (dgasmith) [inline quest/lj.py]: Can you move this to `tests/test_lj.py`?

**Comment by dgasmith** [comment]
@bzhang25 Please ping me when you are ready for this to go in!

**Comment by bzhang25** [comment]
Made all suggested changes, added docs

- [x] add tests

- [x]  put in plot function 

- [ ] make sure function calls to scf and mp2 are correct

**Comment by bzhang25** [comment]
Just added tests/test_lj.py

**Pull request merged**